### DISCUSSION TITLE
add group error

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#91] [Tanya Jawab] [answered]
selamat siang mas rio,

mau tanya apakah ada kendala untuk notifikasi send kirim WA kah ?

sebab saya coba scan ulang berhasil. 
Namun saat cek group syncronize dan send kirim WA selalu muncul pesan seperti ini :

Exception Object ( [message:protected] => Can not fetch HTTP response header. [string:Exception:private] => [code:protected] => 0 [file:protected] =>...

Mohon Bantuannya.

Terima kasih

**Comment by rioastamal** [comment, marked as answer]
@newfaboc tidak ada fungsi add group. Barusan saya coba tidak ada masalah saya dapat melakukan list group dan dapat mengirim pesan ke group. Pastikan koneksi HP stabil dan coba /reconnect dan pastikan WA dan HP menyala waktu melakukan test.

**Comment by newfaboc** [comment]
ok. sudah bisa.
terima kasih banyak mas rio.
